metadataserver/server.go

```diff
@@ -146,6 +146,10 @@ func NewMetaDataServer(ctx context.Context, config types.MetadataServerConfigura
 		signer:                    config.Signer,
 		tokenRequired:             config.RequireToken,
 		xForwardedForBlockingMode: config.XFordwardedForBlockingMode,
+
+		region:             config.Region,
+		availabilityZone:   config.AvailabilityZone,
+		availabilityZoneID: config.AvailabilityZoneID,
```

Comment:
I don't see this ever used?

Reply:
Here: https://github.com/Netflix/titus-executor/blob/124f2faba23f6702a123ad44cc0b8fb43273e3cc/metadataserver/server.go#L419-L474